server/server.go

```diff
@@ -104,6 +104,7 @@ type Server struct {
 	gacc             *Account
 	sys              *internal
 	accounts         sync.Map
+	taccounts        sync.Map // Temporarily stores accounts that are being built
```

Comment:
Threw me, maybe call it tmpAccounts?

Reply:
was my original naming, but since I know you like short names ;-) (will change)